src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java

```diff
@@ -213,19 +213,27 @@ public void run() {
 
     }
 
-    public void initOnNodeStart() {
+    private boolean indexExists(String index) {
+        GetIndexResponse getIndexResponse = client.admin().indices()
```

Comment:
why are we moving to client.admin() from clusterService.state().metaData().hasConcreteIndex(opendistrosecurityIndex)

Reply:
clusterService.state() maintains a cached copy of the cluster state. 

To reproduce the behavior: 
- install plugin 
- start elasticsearch
- restart elasticsearch
- restart elasticsearch

When it starts up, the cluster state is not updated correctly because of which plugin thinks the index does not exist. 
You would see this printed everytime coming from [here](https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java#L223)
```
[2020-06-18T11:19:36,804][INFO ][c.a.o.s.c.ConfigurationRepository] [smoketestnode] .opendistro_security index does not exist yet, so we create a default config
```

Doing the GetIndexResponse would query the ES cluster and get the status whether the index exists or not.

Reply:
Why we dont see the impact is because this log level is debug [here](https://github.com/opendistro-for-elasticsearch/security/blob/master/src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java#L159) 

If you change this to error or info, you would that its throwing error every time in opendistro.